### PR TITLE
Tags property parsing refactoring

### PR DESCRIPTION
**Pull request by kalinchernev**
- Relates to issue #35
- Updates tags property in example files
- Updates expectation swagger specification file to be compliant with swagger validation parser
- Includes tests for ignoring tag property

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -116,17 +117,18 @@ function addDataToSwaggerObject(swaggerObject, data) {
           }
           break;
         }
-        case 'tag': {
+        case 'tags': {
```
Comment (drGrove) [inline lib/index.js]: Add a case for 'tag', don't break and add a `console.warn('tag will be deprecated in v1.0.0, please use tags as it aligns with the swagger v2.0 spec')` then set `propertyName = 'tags';`

Comment (kalinchernev) [inline lib/index.js]: Pushed an update. Used functions to avoid the problem with increasing the complexity.

File: lib/index.js [hunk 2]
```diff
@@ -89,6 +89,42 @@ function objectMerge(obj1, obj2) {
 }
 
 /**
+ * Yields a warning for a given deprecated property.
+ * @function
+ * @param {string} propertyName - The property to warn about.
+ */
+function deprecatedPropertyWarning(propertyName) {
+  if (propertyName === 'tag') {
+    console.warn('tag will be deprecated in v1.0.0');
```
Comment (drGrove) [inline lib/index.js]: There was an error on my part, change v1.0.0 with v2.0.0

Comment (kalinchernev) [inline lib/index.js]: Hi @drGrove could you please add some info about the error?

Comment (drGrove) [inline lib/index.js]: On line 96, replace v1.0.0 with v2.0.0 since we're on v1.4.0 already.

Comment (kalinchernev) [inline lib/index.js]: Done. I thought there is something breaking.
Let me know if anything else is to be handled?

**Comment by kalinchernev** [comment]
@drGrove could you please let me know if there is anything needed before merging this?
Thanks much!

**Pull request merged**